### PR TITLE
Corrected Polish locale for 2.0.

### PR DESCRIPTION
**Pull request by jarosz**
General names.
Fix TODOs - in Polish primary patron is called 'główny' and secondary has no extra word.
Lowercase bishop title.
Re-translate and lowercase priest title.
Lowercase martyr title.
Commas for companions.
Commas for patrons.
Some other typos.

Add missing translations for keys from calendars/poland.js:
* joseph_sebastian_pelczar_bishop
* our_lady_queen_of_poland
* our_lady_help_of_christians
* hedwig_of_poland
* dedication_of_consecrated_churches

### CODE REVIEW

**Comment by emagnier** [comment]
Thanks for your quick contribution! Let's wait for @tukusejssirs comments to then merge this PR asap.

File: src/locales/pl.ts [hunk 1]
```diff
@@ -90,34 +90,34 @@ export default {
     peter_and_paul_apostles: 'Świętych Apostołów Piotra i Pawła',
     presentation_of_the_lord: 'Ofiarowanie Pańskie',
     transfiguration: 'Przemienienie Pańskie',
-    trinity_sunday: 'Niedziela Najświętszej Trójcy',
+    trinity_sunday: 'Najświętszej Trójcy',
```
Comment (tukusejssirs) [inline src/locales/pl.ts:93]: As we try to use the celebration names from the official translation of Roman Missal in the particular vernacular, I think this should be changed to [`Trójcy Przenajświętszej`](https://ministranci.waw.pl/wp-content/uploads/docs/Mszal.pdf#page=83).

Also note that I haven’d proofed any other celebration names. @jarosz, could you please proof the celebration names from the Polish Roman Missal? Ideally from _editio typica tertia_, however, if you can’t access it (or it is not published yet in Polish), you could use _editio typica altera_ or even the above-linked [1986 _editio  typica_](https://ministranci.waw.pl/wp-content/uploads/docs/Mszal.pdf).

File: src/locales/pl.ts [hunk 2]
```diff
@@ -90,34 +90,34 @@ export default {
...
     aloysius_versiglia_bishop_and_callistus_caravario_priest_martyrs:
-      'Świętych Alojzego Versiglii, Biskupa i Kaliksta Caravaria, Kapłana',
+      'Świętych Alojzego Versiglii, biskupa i Kaliksta Caravaria, prezbitera',
     alphonsus_liguori_bishop: 'Św. Alfonsa Marii Liguoriego, biskupa i doktora Kościoła',
     ambrose_of_milan_bishop: 'Św. Ambrożego, biskupa i doktora Kościoła',
     andrew_apostle: 'Św. Andrzeja, Apostoła',
-    andrew_apostle_patron_of_scotland: 'Św. Andrzeja, Apostoła i patrona Szkocji',
-    andrew_bobola_priest: 'Św. Andrzeja Boboli, prezbitera i męczennika',
+    andrew_apostle_patron_of_scotland: 'Św. Andrzeja, Apostoła, patrona Szkocji',
+    andrew_bobola_priest: 'Św. Andrzeja Boboli, prezbitera i męczennika, patrona Polski',
```
Comment (tukusejssirs) [inline src/locales/pl.ts:112]: @jarosz, could you add another key for this celebration? The original key name (without the `patrona Polski`) is also used in Lithuania and Ukraine.

Also, try to localise it other other locales where the ‘base’ celebration (the one without the patronage title) is already present (it should be just a simple copy-paste work).

File: src/locales/pl.ts [hunk 3]
```diff
@@ -171,17 +171,18 @@ export default {
...
     conversion_of_saint_paul_the_apostle: 'Nawrócenie św. Pawła, Apostoła',
     cornelius_i_pope_and_cyprian_of_carthage_bishop_martyrs:
       'Świętych męczenników Korneliusza, papieża, i Cypriana, biskupa',
     cosmas_of_cilicia_and_damian_of_cilicia_martyrs: 'Świętych męczenników Kosmy i Damiana',
     cyril_of_alexandria_bishop: 'Św. Cyryla Aleksandryjskiego, biskupa i doktora Kościoła',
     cyril_of_jerusalem_bishop: 'Św. Cyryla Jerozolimskiego, biskupa i doktora Kościoła',
     cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop: 'Świętych Cyryla, mnicha i Metodego, biskupa',
-    cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_copatrons_of_europe:
-      'Świętych Cyryla, mnicha i Metodego, biskupa, patronów Europy', // TODO: How to say `secundary patrons` in Polish?
+    cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_copatrons_of_europe:
```
Comment (tukusejssirs) [inline src/locales/pl.ts:182]: Could you remove the trailing whitespace please? Even better: could you install and use [`.editorconfig`](editorconfig.org) please? :smiley:

File: src/locales/pl.ts [hunk 4]
```diff
@@ -293,7 +296,7 @@ export default {
     martin_de_porres_religious: 'Św. Marcina de Porres, zakonnika',
     martin_i_pope: 'Św. Marcina I, papieża i męczennika',
     martin_of_tours_bishop: 'Św. Marcina z Tours, biskupa',
-    martin_wu_xuesheng_and_companions_martyrs: 'Świętych męczenników Marcina Wu Xueshenga i Towarzyszy',
+    martin_wu_xuesheng_and_companions_martyrs: 'Świętych męczenników Marcina Wu Xueshenga, i Towarzyszy',
```
Comment (tukusejssirs) [inline src/locales/pl.ts:299]: I am not sure of the more recent editions, however, in the [_editio typica_](https://ministranci.waw.pl/wp-content/uploads/docs/Mszal.pdf#page=850), they don’t use a comma before `i Towarzyszy`.

**Review by tukusejssirs — CHANGES_REQUESTED** [review]
I’ve finished my ‘quick’ review of this PR. I haven’t gone any deeper (like comparing all the key values with the official Polish translation of Roman Missal); I just simply read all the changes.

Note that I am not sure that all my comments are 100 % correct; do check upon it please.

File: src/locales/pl.ts [hunk 5]
```diff
@@ -388,8 +393,8 @@ export default {
     sylvester_i_pope: 'Św. Sylwestra I, papieża',
     teresa_benedicta_of_the_cross_stein_virgin: 'Św. Teresy Benedykty od Krzyża, dziewicy i męczennicy',
     teresa_benedicta_of_the_cross_stein_virgin_copatroness_of_europe:
-      'Św. Teresy Benedykty od Krzyża, dziewicy, męczennicy i patrona Europy', // TODO: How to say `secundary patroness` in Polish?
-    teresa_of_jesus_of_avila_virgin: 'Św. Teresy od Jezusa, dziewicy i doktora Kościoła', // TODO: Localise as `Saint Teresa of Jesus of Ávila, Virgin and Doctor of the Church`
+      'Św. Teresy Benedykty od Krzyża, dziewicy i męczennicy, patronki Europy',
+    teresa_of_jesus_of_avila_virgin: 'Św. Teresy od Jezusa, dziewicy i doktora Kościoła',
     therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin:
       'Św. Teresy od Dzieciątka Jezus, dziewicy i doktora Kościoła',
     thomas_apostle: 'Św. Tomasza, Apostoła',
```
Comment (tukusejssirs) [inline src/locales/pl.ts:400]: @jarosz, although it is not yet in the `_F-TypeScript` nor `dev` branch, `wenceslaus_i_of_bohemia_martyr_patron_of_czech_nation` will be added as soon as #248 will be merged. Please, review my Polish translation of the key below (more info in [this comment](https://github.com/romcal/romcal/pull/248#discussion_r518015314)). Thanks!

```json
pl : 'Św. Wacława, męczennika i patrona narodu czeskiego',
```

**Review by tukusejssirs — COMMENTED** [review]
@jarosz, I have added a review comment long time ago [some day(s) after you created this PR), but I forgot to submit the review and thus my comment did not show up.

The comment is not a blocker, just an additional note. :smiley:

@emagnier, this PR is ready to be merged from my point of view. :smiley:

**Comment by emagnier** [comment]
@tukusejssirs: as you are suggesting, we could merge this PR, since there is already quite a lot of improvements here. The remaining notes could be addressed later. Are you still ok with that?

**Comment by tukusejssirs** [comment]
> @tukusejssirs: as you are suggesting, we could merge this PR, since there is already quite a lot of improvements here. The remaining notes could be addressed later. Are you still ok with that?

Indeed. I’ll find some time to create an issue from these comments in an hour or so.

**Pull request merged**